src/Tizen.NUI/src/internal/PageTurnView.cs

```diff
@@ -424,107 +411,11 @@ public static PageTurnView GetPageTurnViewFromPtr(global::System.IntPtr cPtr)
             return ret;
         }
 
-
-        /// <since_tizen> 3 </since_tizen>
-        public new class Property : global::System.IDisposable
+        internal new class Property
```

Comment:
please make this HiddenAPI (not internal) ; please add [EditorBrowsable], [Obsolete] attributes like others done.